docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>These plots great!</p><p> </p><p>Any chance we can make them a bit bigger? They feel kind of cramped.</p><p> </p><p>If it's a pain, don't worry about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>I changed to svg and ran <code>alt.renderers.set_embed_options(width=500, height=600)</code>,which I think looks a little better.  </p>